src/meta.ts

```diff
@@ -37,6 +37,10 @@ export class Meta {
       context.ref = `refs/pull/${context.payload.number}/merge`;
     }
 
+    if ((/pull_request/.test(context.eventName) || /pull_request_target/.test(context.eventName)) && context.payload?.pull_request?.head?.sha != undefined) {
```

Comment:
This appears to check that substring “pull_request” appears anywhere in the event name. Maybe the intention was to check if the name starts with a certain substring? Or matches exactly if that’s the entire event name?